### PR TITLE
Groundside Roundstart Gameplay Tweak

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Shakes up roundstart gameplay for everyone groundside by implementing a bunch of wild changes:
- Survivors now have role specific skills, meaning one survivor is weaker, and need allies to cover their deficiencies.
- More survivors spawn, with a minimum of 4 (up from 2) and a maximum of 12 (up from 8) slots. Additionally, the calculation has been changed to 1 per 8 people (up from 1 per 11).
- There is a 25% chance for a second Synthvivor slot to be opened, and the special spawns like CLF ship now spawn them as a CLF synth instead of a human.
- Added a PMC synth.
- Survivors now have a 60% chance not to spawn with a primary weapon, and the weapons they do spawn with are hunting rifles, M16s, Mar30s, and MP5s.
- Survivors no longer have marine IFF.
- Survivors now spawn with a medium rather than a large stack of metal.
- Xeno castes can now only evolve after a certain amount of time has passed, generally, the more dangerous the caste, the longer it takes until they're available.
- Xenos now gain "bonus evolution" when they gain evolution past their evolution max. This bonus evolution is carried on to their next xeno when they evolve, meaning staying T1 for a while doesn't waste your T2 evo.
- Larva evo is no longer special, and works as all other xeno evo does.

NOTE: The opinions of this PR is my own and didn't have any Dev team discussion (yet). This is a big shake-up, so not every change is set in stone, any feedback on this PR will be taken into account to fine-tune it.

## Why It's Good For The Game

Current roundstart gameplay is a mad rush to get everything perfectly in place because xenos and survivors start off strong, this change nerfs them both and should slow down the pace.

## Changelog

:cl:
balance: Survivors now have role specific skills, meaning one survivor is weaker, and need allies to cover their deficiencies.
balance: More survivors spawn, with a minimum of 4 (up from 2) and a maximum of 12 (up from 8) slots. Additionally, the calculation has been changed to 1 per 8 people (up from 1 per 11).
balance: There is a 25% chance for a second Synthvivor slot to be opened, and the special spawns like CLF ship now spawn them as a CLF synth instead of a human.
add: Added a PMC synth.
balance: Survivors now have a 60% chance not to spawn with a primary weapon, and the weapons they do spawn with are hunting rifles, M16s, Mar30s, and MP5s.
balance: Survivors no longer have marine IFF.
balance: Survivors now spawn with a medium rather than a large stack of metal.
balance: Xeno castes can now only evolve after a certain amount of time has passed, generally, the more dangerous the caste, the longer it takes until they're available.
balance: Xenos now gain "bonus evolution" when they gain evolution past their evolution max. This bonus evolution is carried on to their next xeno when they evolve, meaning staying T1 for a while doesn't waste your T2 evo.
balance: Larva evo is no longer special, and works as all other xeno evo does.
/:cl:

### CODE REVIEW

**Comment by Listaless** [comment]
Heavily against most of the changes, I'll state reason for each one I'm against below(most of this is survivor related as that's what the changes are impacting)

 Survivors now have role specific skills: Personally I like being able to pick my playstyle as a survivor, I like doing engineering stuff so If I don't spawn as a engineer I'm completely boned, could be counteracted by allowing people to spawn with a choose skill set, if you really wanted to push this through. (My largest complaint, I don't mind lacking skills if I can pick the skill/skills I really want.)
 
 Survivors now have a 60% chance not to spawn with a primary weapon: Your going to get rushed by runners, spawning with weapons was added to mitigate the lower evo times because xeno's like faster evo and no maturity.(Highly depends on belows timers)
 
 Xeno castes can now only evolve after a certain amount of time has passed: anyone mind replying to my comment with the times for this, hard to judge without knowing the timers.
 
 Otherwise I don't mind trying this out if it slows down the pace of the roundstart and allows for more prep time.

**Comment by gnorse** [comment]
> Maybe consider small injuries when you spawn (and still a decent chance to just be okay). Maybe you have a fracture thats lingering and pre-survivor you were trying to find medical for spints. Maybe you burnt your eyes welding without goggles and have the first phase of blindness. I dunno - just basic variety of things that can contribute to lore of your character that round and should be resolvable if you work with your survivor friends.

if you spawn with a broken leg/foot or eye damage as a survivor you are absolutely 100% without a doubt fucked, no matter what equipment/skills you have access to.

**Comment by c4xmaniac** [comment]
I like the changes, though I think all survivors should have medical 1 allowing them to use auto injectors because being able to heal is a necessity. This should come with a mapping rework for the maps adding more weapons/ammo though (the rng spawn chance ones stan added). If you have 12 survivors on LV with only 60% of them having guns, not everyone will have a gun. The soviet one man having the gun and the other having the ammo is funny, but not actually fun to play. I'd be down to be your little mapping gremlin for that if you were interested. If you did add extra weapon spawns you could reduce the default spawn to a pistol/rifle % chance and reduce the % chance for starting with a weapon as well.

Something minor to consider is back in the day there used to be lore for when you spawned as survivor telling you what you were doing before the xeno outbreak. Like "You and X other surv were playing basketball when a drone showed up and killed a dude on the court, you then went your separate ways." Its not really in the scope of this PR, but it would be a cool thing to have back if you were interested at some point.


> Maybe consider small injuries when you spawn (and still a decent chance to just be okay). Maybe you have a fracture thats lingering and pre-survivor you were trying to find medical for spints. Maybe you burnt your eyes welding without goggles and have the first phase of blindness. I dunno - just basic variety of things that can contribute to lore of your character that round and should be resolvable if you work with your survivor friends.

Prespawn injuries have largely been phased out because they're not that fun. Some of the roles still have them, the 1% chance BR PMC spawn has it for instance, but that role gets access to insanely OP gear so its understandable to make its start extra hard. The best example of why the prespawn injuries aren't a good idea for normal roles is the LV Cabin spawn which used to spawn you injured, not only is the spawn horrible (assured death), but the rng would remove both of your legs so you would spawn unable to even turn your flashlight on.

**Comment by SideCat** [comment]
Some changes should be reconsidered for balanced and more enjoyable gameplay:
- Survivors now have role specific skills, meaning one survivor is weaker, and need allies to cover their deficiencies.
 Issue: Roles such as truck driver on BR, cargo tech on LV, humanitarian overlooker on shivas, CL on any map and possible more roles that I've missed out will serve no contribution to a survivor team assuming firearm skill will be reduced and shotguns will be taken away from spawn pool. 
 Solution: remain firearm skill untouched as is or compensate for given roles an useful skill similary how researchers have medical skills.
- More survivors spawn, with a minimum of 4 (up from 2) and a maximum of 12 (up from 8) slots. Additionally, the calculation has been changed to 1 per 8 people (up from 1 per 11).
 Issue: currently max surv cap is 8 and rarely it's being filled out by players. Adding more slots to an already very beginner unfriendly role with on top upcoming role nerf PR will turn out fruitless compensation for giving survivors role nerfs. Not to mention if by chance all 12 survs will be spawned there won't be enough ammo/weapons to go by for map loot were all planned with 8 surv max cap.
 Solution: more surv slots are uneeded, rarely they're being filled out so another compensation would be better. Unfortunately the only compensation I can think of for now would be less heavily nerfed roles even though that's the whole purpose of this PR.
- Survivors now have a 60% chance not to spawn with a primary weapon, and the weapons they do spawn with are hunting rifles, M16s, Mar30s, and MP5s.
 Issue: removing shotgun from the spawn pool on firearm nerfed survivors could make them defenceless against swarm of runners. There are only few maps: BR/Trijent/Shivas that has a lots of shotguns to go by for planned 12 surv slots. A lot of fighting could occure over 1 easily accessible shotgun that can spawn on some maps. Others are either required engineering skills or they're too close to hive. Why shotguns are so favourable? Because it's good against close combat xenos and it was good for survivors when they previously had only firearm 1 since accuracy is not heavily required to be useful.
 Solution: don't remove shotgun from spawn pool or compensate by putting more shotguns around the map.

tl:dr not enough compensation for big surv nerfs. Sounds good on paper, terrible in action.

**Comment by MusselMain** [comment]
> Some changes should be reconsidered for balanced and more enjoyable gameplay:
> 
> * Survivors now have role specific skills, meaning one survivor is weaker, and need allies to cover their deficiencies.
>   Issue: Roles such as truck driver on BR, cargo tech on LV, humanitarian overlooker on shivas, CL on any map and possible more roles that I've missed out will serve no contribution to a survivor team assuming firearm skill will be reduced and shotguns will be taken away from spawn pool.
>   Solution: remain firearm skill untouched as is or compensate for given roles an useful skill similary how researchers have medical skills.
> * More survivors spawn, with a minimum of 4 (up from 2) and a maximum of 12 (up from 8) slots. Additionally, the calculation has been changed to 1 per 8 people (up from 1 per 11).
>   Issue: currently max surv cap is 8 and rarely it's being filled out by players. Adding more slots to an already very beginner unfriendly role with on top upcoming role nerf PR will turn out fruitless compensation for giving survivors role nerfs. Not to mention if by chance all 12 survs will be spawned there won't be enough ammo/weapons to go by for map loot were all planned with 8 surv max cap.
>   Solution: more surv slots are uneeded, rarely they're being filled out so another compensation would be better. Unfortunately the only compensation I can think of for now would be less heavily nerfed roles even though that's the whole purpose of this PR.
> * Survivors now have a 60% chance not to spawn with a primary weapon, and the weapons they do spawn with are hunting rifles, M16s, Mar30s, and MP5s.
>   Issue: removing shotgun from the spawn pool on firearm nerfed survivors could make them defenceless against swarm of runners. There are only few maps: BR/Trijent/Shivas that has a lots of shotguns to go by for planned 12 surv slots. A lot of fighting could occure over 1 easily accessible shotgun that can spawn on some maps. Others are either required engineering skills or they're too close to hive. Why shotguns are so favourable? Because it's good against close combat xenos and it was good for survivors when they previously had only firearm 1 since accuracy is not heavily required to be useful.
>   Solution: don't remove shotgun from spawn pool or compensate by putting more shotguns around the map.
> 
> tl:dr not enough compensation for big surv nerfs. Sounds good on paper, terrible in action.

This change is a interesting Idea, but it definitely needs some alteration.
Removing primaries and especially all the shotgun options from survivors could end up scaring away new players from choosing survivor as they might be too intimidated by the role, the sidearms survivors spawn with range from 'functional' to 'useless' and at their best MIGHT be able to kill a runner (However this is because most sidearms aren't designed to be useful beyond fighting one T1), But if this change is added, survivors sidearm pool needs to be altered to remove (or buff) the KT-42, and the CMB Spearhead Autorevolver, as starting with either of those will actually be a death sentence, also sidearms need to be added to the loot pool of Prisoners on FIorina as they ususally start without one.   

The way I think this COULD work, is if there's a bit more of a compromise, just because there's more survivor slots does not mean they will be filled, I think the chance of some survivors spawning without a primary should be affected by population, if survivor numbers are low (and it isn't low-pop), there should be a higher chance of survivors starting with a primary, this way if there's a round where only two survivors spawn in a 150+ player game, they aren't completely fucked, and on the opposite end, if there's 12 survivors in a lower pop round they don't all come armed to the teeth like a ERT. 

I also wanted to mention that maybe the number of skill points should be adjustable based on what role you are, some roles such as Standard Colonist, or Corporate Liaison lack many skills that other survivors have, and therefore should have more survivor points to add to their skills (unless you want to make them the Hide-in-a-locker roles).

To conclude all I've written, this is a interesting idea, however survivors need their skills altered (especially CL), the chance that survivors spawn without a primary should be affected by the number of survivors in the round (If possible) as too few survivors in High pop, without primaries will be very weak, and too many in low pop all armed with primaries, could be too strong.

**Comment by FrogZeke** [comment]
There are a few points I would like to address here.

Lore Accurate Survivors:  If the goal is to create lore accurate survivors, sources should be used.  In 'Newt's Tale', that M-37 is the STANDARD weapon used by survivors, with pistols being a backup, this is depicted several times in the first issue.  In the novelization of 'Newt's Tale', 'River of Pain', not only are shotguns and pistols the standard issue for colony defense, there is a marine detachment on Acheron led by one Captain Brackett, he escapes the colony and his debrief is presented in the 'The Weyland-Yutani Report'.  Also, many of the ducts through colonies are large enough for humans to crawl through them, with small children being able to run through them.  This is depicted in Alien and Aliens, several comics, video games, and a few novels.

Survivors now have role specific skills:  This is true, but the reality of the player base needs to be taken into account.  While survivor regulars generally know that working as a group improves their odds of surviving or being revived, there are still some that choose hiding and not communicating.  If the balance is established for a group of survivors, hiding needs to be taken into account.  A 12-survivor round with 6 locker hiders is a 6-survivor round, and incredibly imbalanced based on random skills and the number of xenos.  When you account for hiding, it may made a round imbalanced in favor of the survivors.  For the sake of balancing, allow locker hiding to be against the rules, so it can be ahelped and bans applied.

Survivors now have a 60% chance not to spawn with a primary weapon, and the weapons they do spawn with are hunting rifles, M16s, Mar30s, and MP5s:  The only way to fight stun combat is using stun combat in turn.  None of these weapons take this into account, and is just asking for survivors to die in non-caded situations.  Shotguns are cheap to make, costing 1/4 of sub-machine guns and assault rifles.  This would weigh survivor weapons towards shotguns, with sub-machine guns and assault rifles being the rarity.  Some maps have enough weapons to cover survivors, but rarely is this something that can be done without roaming, which is dangerous now, but will be more so with the skill changes.

Even with all that said, I am not against attempting these changes in this merge as a week long test.  Pal Johnson collected data on survival rates, captures, and xeno deaths to back up his survivor changes.  It may be a case of a merge of this complexity to require map changes such as some tunnels being removed to prevent xenos rushes, or potentially adding xeno base defense to prevent a survivor rush.  I do not know, but I know randomness makes things very difficult.

**Comment by Katskan** [comment]
Role specific skills might be good if you applied the same type of role selection as ERTs have. Let people select yea/nay on different surv types, then only role for those positions. E.g. I personally would only check doctor and scientist, then when I roll for survivor, I personally will only spawn if I win the roll for the doctor and scientist roles that are selected for that round. 

Another thing, with the removal of basic medical skill from most survivors, there needs to be guaranteed at least 2 roles spawning with medical skill. If nobody has any medical skill, the survivors are totally screwed. And if there's only one doctor/scientist, and something happens to them (it probably will) then everyone is screwed. It's one thing to make survivors rely on each other for tasks, and another to completely remove their ability to heal and give them a super obvious Achilles heel that xenos can target to eliminate the group with ease.

**Comment by Listaless** [comment]
Right going to say this now, removing ENGI 2 is not a wise idea, Why you might ask?
First: you can't hack airlocks, This is fine (Personally: No it an't give me my meme doors)
Two: you can't repair generators: Technically this is fine right, no because of the below reason
Three: you can't open up APC's to switch out power cells, WHICH MEANS.....drumroll

because MOST maps start unpowered you can't vend ANYTHING, no medical supply's, no armour from a sec vendor, no food, nothing, no make matters worse spawns like engineer or doctor are still random and even if they weren't this is still a horrible change.

I can deal with spawning without a gun
I can deal with lack of IFF(only really apply's to like three maps for early game)
but I an't dealing with lacking a core part of survivor gameplay.

**Comment by SilverBerryWave** [comment]
Increasing surv numbers and blocking xenos from actually evoing to the caste they want to play makes for a pretty horrible xeno experience.

**Comment by Mister-moon1** [comment]
this has just led to survivors having 0 quarrels getting all the juicy loot off the map and rushing xenos before they even have t2s, literally survs are rushing hive and xenos cant do anything especially since runners and lurkers dont come in until 10 mins after round start

**Comment by Carnage068** [comment]
Update the changelog for the xeno evolution timers, it's way off.

**Comment by JohnDoe50974** [comment]
My opinion on the TM after observing it in action for cca. 5 rounds.


-  The current roundstart gameplay loop seems to be the best on LV, especially when fog prevents access to the xeno side of the map.
-  Civillian/survivor weapons will need a balance pass down the line. The power of some of them (like the MAR series rifles) has become evident, as they were clearly balanced to account for the fact survivors will generally be stuck fighting hopeless battles 2 minutes into the round, as was the case beforehand. Right now this may lead to situations where survivors are containing xenos and make them unable to spread beyond their hive location and the immediate perimeter, which has a negative impact on xeno gameplay experience - if this happens drones cant do their job until T3s can evolve to overwhelm the survivors.
- Adding onto above, I recommend reducing or returning the survivor cap to its previous value. I believe the current cap would just continue to facilitate this behavior on maps littered with firearms whenever the maximum amount of survivors spawn.
- Map loot in general will need to be looked at. Annex in particular with good random loot RNG may allow survivors to dispatch a sizeable amount of xenos in areas where previously such holdouts would be a death sentence for survivors instead. But that might broaden the scope of this already massive PR too much :)
- The evolution time limits on castes all seem fine to me. Again, the biggest potential gameplay issue right now seems to be the possibility of survivors preventing drones from weeding outside their hive, since right now xenos cannot protect drones as they work.

**Comment by The-Sun-In-Splendour** [comment]
cm devs on their way to explain why forcing xenos to afk in hive for longer will make the roundstart more fun

**Comment by c4xmaniac** [comment]
I’ve finally gotten to watch and play surv with these changes and here are my thoughts:

Some people are purposefully trying to abuse the higher surv count plus longer xeno evo to abuse the xenos and wipe the hive pre drop. I think this is solvable by just returning the previous rules of attacking the hive (you cannot attack the hive until the queen is mature, with your times I think that’s right about when the hive gets t2s) or extending it for longer. You can also decrease weapon spawns as surv and add more stuff around the maps as I said previously, this way there’s less time of shooty shooty and wall off bug and more oh fuck I need a gun and ammo.

Survs spawning with 75% hp or there abouts is kind of lame. You don’t get access to pain meds by default, and you’re forced to expend 1/4 of your starting meds (tricked pills) from the get go. I get what you’re going for with it, but it doesn’t land that well game play wise. This is exacerbated when every single surv spawns with exactly the same HP so when you have a medhud everyone is hurt the exact amount. I’d prefer not having the damage at all, but if you’re going to add it some variability would be nice. Just remember survs only default medicine is a 4 pill packet of tricord - no pain pills. 

You shouldn’t start with a MAR series gun, it should be hunting rifle, mp5, f2000, m16, or HG. The Mars are weak vs runners, but strong vs virtually every other caste. The f2000 is terrible by the way, I don’t know why it’s as rare as it is in game, but it’s still better than the hunting rifle. I think the mp5/m16/f2000/hunting rifle/HG provides a better mix of starting guns without needing to go through and rebalance half of the weapons groundside, especially when those balance changes would carry over into ERT and HVH.

I got to see the double surv synth spawn, I think it’s going to be very rare (not a ton of people roll surv synth anyways) but it was really neat to see. I hope that we’ll see surv synths working against one another in the future due to siding with different factions and having different agendas. Lots of RP opportunities.

I haven’t gotten to play xeno with the changes, but I doubt there’s any real impact. Some people complained about afking in the hive, but that’s literally what I do as xeno anyways so there’s no change. I do think it’s movie like af to have the drone castes slither out first to lay the groundwork for the other xenos to occupy the colony, and it provides a kind of rolling storm. It might be cool to add some variability to these numbers so you never know as a surv when X caste is coming. Maybe this round runners can’t come till 10min, but warriors can come at 6min. It could provide more unique rounds and with the evo tick change, people who still don’t want to interact with survs aren’t affected. It’ll get people to surv hunt as different castes which could be cool. 

I also think that xenos are going to develop new strategies and metas to deal with survs. I saw Chi clown car the survs on a BR round and cap them, obviously there’s big risk (embarrassing queen/t3 deaths) but holding back your xenos till you can clown car your way through provides opportunities for big cap gains with 12 survs. I think they should give it time, xeno players are very cunning and they’ll find ways to make the most out of higher surv numbers, especially when most surv players aren’t particularly good.

**Comment by Lumdor** [comment]
Horrible change, no other explanation is needed just play the game and see for yourself :).

**Review by stanalbatross — CHANGES_REQUESTED** [review]
- slots increase is certainly too much
- extra synth surv slot is weird. 25% of rounds have a massive marine buff in them? Should be removed.

**Comment by Waseemq1235** [comment]
> Horrible change, no other explanation is needed just play the game and see for yourself :).

istg geeves is like that one /tg/station dev that kept PRing meme changes to the game

**Comment by aceroyuno** [comment]
The game is already lightning fast as is. Please explain how making rounds take less time to complete is an improvement. It very well could be. But overall this change feels very biased towards survs, who as it is already, have no gameplay objectives. Please take a moment to think about the bigger picture of what you are doing. If you want xenos and survs nerfed, you need to nerf the ENTIRE marine corp just to compensate it.

File: code/modules/mob/living/carbon/xenomorph/castes/Runner.dm [hunk 1]
```diff
@@ -24,6 +24,8 @@
 
 	heal_resting = 1.75
 
+	minimum_evolve_time = 8 MINUTES
```
Comment (JohnDoe50974) [inline code/modules/mob/living/carbon/xenomorph/castes/Runner.dm:27]:
```suggestion
	minimum_evolve_time = 5 MINUTES
```

File: code/modules/mob/living/carbon/xenomorph/castes/Sentinel.dm [hunk 2]
```diff
@@ -26,6 +26,8 @@
 
 	behavior_delegate_type = /datum/behavior_delegate/sentinel_base
 
+	minimum_evolve_time = 5 MINUTES
```
Comment (JohnDoe50974) [inline code/modules/mob/living/carbon/xenomorph/castes/Sentinel.dm:29]:
```suggestion
	minimum_evolve_time = 4 MINUTES
```

File: code/modules/mob/living/carbon/xenomorph/castes/Defender.dm [hunk 3]
```diff
@@ -23,6 +23,8 @@
 	tackle_min = 2
 	tackle_max = 4
 
+	minimum_evolve_time = 5 MINUTES
```
Comment (JohnDoe50974) [inline code/modules/mob/living/carbon/xenomorph/castes/Defender.dm:26]:
```suggestion
	minimum_evolve_time = 4 MINUTES
```

File: code/modules/mob/living/carbon/xenomorph/castes/Spitter.dm [hunk 4]
```diff
@@ -29,6 +29,8 @@
 	tacklestrength_min = 4
 	tacklestrength_max = 5
 
+	minimum_evolve_time = 8 MINUTES
```
Comment (JohnDoe50974) [inline code/modules/mob/living/carbon/xenomorph/castes/Spitter.dm:32]:
```suggestion
	minimum_evolve_time = 7 MINUTES
```

File: code/modules/mob/living/carbon/xenomorph/castes/Warrior.dm [hunk 5]
```diff
@@ -27,6 +27,8 @@
 
 	heal_resting = 1.4
 
+	minimum_evolve_time = 10 MINUTES
```
Comment (JohnDoe50974) [inline code/modules/mob/living/carbon/xenomorph/castes/Warrior.dm:30]:
```suggestion
	minimum_evolve_time = 7 MINUTES
```

File: code/modules/mob/living/carbon/xenomorph/castes/Lurker.dm [hunk 6]
```diff
@@ -23,6 +23,8 @@
 
 	heal_resting = 1.5
 
+	minimum_evolve_time = 8 MINUTES
```
Comment (JohnDoe50974) [inline code/modules/mob/living/carbon/xenomorph/castes/Lurker.dm:26]:
```suggestion
	minimum_evolve_time = 7 MINUTES
```

File: code/modules/mob/living/carbon/xenomorph/castes/Burrower.dm [hunk 7]
```diff
@@ -34,6 +34,8 @@
 	widen_cooldown = 70
 	tremor_cooldown = 450
 
+	minimum_evolve_time = 8 MINUTES
```
Comment (JohnDoe50974) [inline code/modules/mob/living/carbon/xenomorph/castes/Burrower.dm:37]:
```suggestion
	minimum_evolve_time = 7 MINUTES
```

File: code/modules/mob/living/carbon/xenomorph/castes/Carrier.dm [hunk 8]
```diff
@@ -37,6 +37,8 @@
 	hugger_delay = 20
 	egg_cooldown = 250
 
+	minimum_evolve_time = 6 MINUTES
```
Comment (JohnDoe50974) [inline code/modules/mob/living/carbon/xenomorph/castes/Carrier.dm:40]:
```suggestion
	minimum_evolve_time = 5 MINUTES
```

**Review by JohnDoe50974 — COMMENTED** [review]
The current minimum evolve times are too lenient. Try changing it per my suggestions together with the changes stan requested and let's see how that plays out.

**Comment by JohnDoe50974** [comment]
The single hardest PR of 2022, rest in power.

**Comment by Mirvra** [comment]
Personally think the ''Bonus evolution'' thing was fairly ok.
And diversifying the survivors seemed ok. Though I'm not a Survivor player so I'll leave that up to them to comment on.

The two issues I saw were:
1, The fact that both these things are in the same PR, instead of being two separate ones.
2, That the PR basically gave survivors/early marines free reign of majority of the map with survivors capable of literally fixing power and blowing all podlocks before xenos could even get a Sentinel out.


Addendum: The one thing I did like about the evo timer locks though was that it did actually let me go Drone, help build and weed for a bit, and then de-evo and instantly have enough evo to only be slightly behind getting a T2 Spitter slot.

Addendum 2: Noticed that as T3 you didn't earn more ''bonus evolution'' BUT you still had all the bonus you had accrued as T2, so you could technically de-evo and evo a bunch in rapid succession.

**Comment by PalJohnson** [comment]
Hey Geeves I wanted to give you some feedback in regards to the update here. 

I have to say in general I liked the update and I hope you get to read this through all the salt being thrown your way. Overall the survivor meta changed a fair bit and I wanted to break down my experience of having players over a dozen rounds while the TM was up. 

In short this did a hard reset to survivor style of gameplay and I enjoyed it a lot. With the skill decrease across the board, I found that survivors were more likely to stick together to accomplish certain objectives like restoring power, hacking, and overall savaging for supplies. Supplies on the map haven't changed so it strained the economy of load-outs for certain greedy individuals but overall we had just enough to pass around. 

I expected holdouts to be greatly diminished in power level however with enough savaging given the allotted time I would say we were still able to have a semi-decent defensive position by the time xenos attacked us. I was also collecting data during the tm and found the average engagement time for xenos v survivors was about 5-10min average. I think time is fine as is. I recommend xenos travel in packs as drones, Hive lords, and defenders as they are the earliest evo they can select.

The evolution time changed the ground side gameplay for xenos as previous combat classes were inaccessible (like the runner) which could disrupt survivor loops straight out the gate due to speed and lack of rifles. Xenos tended to stay in hive entirely for evo until they got their desired class then attack. Surprisingly even then they choose to sit in the hive leaving drones and hive lords unprotected in the colony resulting in survivors gaining a lot of ground that otherwise would have been weeded normally. I think its funny that the new evo system means you don't lose points for evolution when evolving like a reserve. I would say over half of the xenos who joined the game during TM choose to spend more time as Larva than they would drone or hive lord support classes.

When Xenos did engage with survivors it was usually warrior, lurker, and T3s which happened around 12:15 and compounded the siege (if holdout was constructed) down to about 10-15min ish prior to marine arrival. We had prepared fortifications by that point but even then not all the barricades in the world could stop T3s so we had to abandon them most of the time. At times it was a little boring as people who normally hunted survivors had no experience with classes like defender, sentinel, or drones so they afked in hive until they had EVO points to go to the desired class.

In terms of gear, I recommend removing the tools pouch as most survivors don't have hacking skills and it is more of a relic of the previous style of gameplay this MR is looking to avoid. I would instead replace it with a medium pouch or a first aid pouch with basic supplies except on the engineer who can still use tools. The tools pouch is kind of a dead slot with nothing of value other than perhaps facilitating lights onto guns. Add screwdriver to right ear as it is still needed in current meta. 

As per synth survivor (having played over a dozen rounds) I can say I didn't see any double synths. Its on the same level of rarity as clf survivors at bout 25%. I think its fine as is and I would like to remind people that synths don't have to align with marines per their roleplay standards. Some synths just do their own thing during the round and don't even engage with survivors or marines.

As per mapping changes perhaps just add a little more ammo for weapons you included in the weapon pool. Once you run out of ammo its over as you have no other source of ammo to refill magazines. I would also consider adding a shotgun to the weapon pool or add more of them across the map in random locations Not having a shotgun wasn't the biggest deal in the world but it was nice to have. Also, try buffing colony pistols. 357 does less damage than the 9mm and it would be nice to not just instantly dump the pistol roundstart.

Survivor numbers really didn't need a bump (in the previous meta) but with the changes, I think its fine. We lost a lot of stuff but in the process gained 2 more individuals. You still need to be fairly robust and now have to rely on others to get the job done. Survivor murder balls already existed prior now you just have more people fighting or holding a spot.

Im kind of shocked people feel it's biased towards survivors. Survivors got a lot of nerfs in the process of making this PR with the only upside being more people to compensate. Loadout nerfs, weapon nerfs, skills nerfs and yet so much salt saying survivors are OP. This update made it so survivors needed to stick together to live were as previously survivors could if they wanted too roam alone and be somewhat robust. Robustness isn't the only thing you need to survive its teamwork. This was an excellent update and again keep up the good work.

**Comment by Katskan** [comment]
I had an absolute blast with this. It opened up RP opportunities to a massive extent for survivors, I saw people playing as reporters pretending to be recording a video for the news, people trying to figure out sleeping arrangements and food, heating and cooling, rather than just spending every breath on fighting off a swarm of runners 2 minutes into a round. Survivors actually got to be survivors with valuable skills and a group mentality, rather than just focusing on being Robust Man Robusto III, and that brings with it the chance to slow down a bit and vibe, rather than being PFC+ with an early drop to the AO.
